src/api/posts/package.json

```diff
@@ -25,6 +28,8 @@
     "normalize-url": "6.1.0"
   },
   "devDependencies": {
+    "@senecacdot/eslint-config-telescope": "1.0.0",
```

Comment:
We just updated eslint-config-telescope to v1.1.0 in #3267 and many of the eslint configurations were refactored in #3253. To pick up the latest updates which addresses the no-shadow error, change the `1.0.0` to `1.1.0`